internal/telemetry/sensitivemetadataallowlist/sensitivemetadataallowlist.go

```diff
@@ -37,39 +37,123 @@ func AllowedEventTypes() EventTypes {
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.chatResponse",
+			Action:  "%",
+			AllowedPrivateMetadataKeys: []string{
+				"chatModel",
+			},
+		},
+		// The 'chatModel' key is included for feature:'cody.chatResponseNew' events to provide
+		// customers with valuable chat-model specific insights from the analytics we offer.
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.chatResponseNew",
+			Action:  "%",
+			AllowedPrivateMetadataKeys: []string{
+				"chatModel",
+			},
+		},
+		// The 'model' key is included for feature:'cody.command.doc' action: 'executed' events to provide
+		// customers with valuable chat-model specific insights from the analytics we offer.
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.command.doc",
+			Action:  "executed",
+			AllowedPrivateMetadataKeys: []string{
+				"model",
+			},
+		},
+		// The 'model' key is included for feature:'cody.command.edit' action: 'executed' events to provide
+		// customers with valuable chat-model specific insights from the analytics we offer.
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.command.edit",
+			Action:  "executed",
+			AllowedPrivateMetadataKeys: []string{
+				"model",
+			},
+		},
+		// The 'model' key is included for feature:'cody.command.fix' action: 'executed' events to provide
+		// customers with valuable chat-model specific insights from the analytics we offer.
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.command.fix",
+			Action:  "executed",
+			AllowedPrivateMetadataKeys: []string{
+				"model",
+			},
+		},
+		// The 'model' key is included for feature:'cody.command.test' action: 'executed' events to provide
+		// customers with valuable chat-model specific insights from the analytics we offer.
+		// Access to this information can help customers determine which models best suit their specific use cases
+		EventType{
+			Feature: "cody.command.test",
+			Action:  "executed",
+			AllowedPrivateMetadataKeys: []string{
+				"model",
+			},
+		},
 	)...)
 }
 
 type EventTypes struct {
 	types []EventType
-	// index of '{feature}.{action}:{allowedfields}' for checking
+	// index of '{feature}:{action}:{allowedfields}' for checking
 	index map[string][]string
 }
 
 func eventTypes(types ...EventType) EventTypes {
 	index := make(map[string][]string, len(types))
 	for _, t := range types {
-		index[fmt.Sprintf("%s.%s", t.Feature, t.Action)] = t.AllowedPrivateMetadataKeys
+		key := fmt.Sprintf("%s.%s", t.Feature, t.Action)
```

Comment:
In hindsight this index should use another character that isn't a legal feature/action character, to avoid situations where:

```
feature: a.b, action: c
feature: a, action: b.c
```

resolves to the same entry. the `/` character might be a better one to use here:

```suggestion
		key := fmt.Sprintf("%s/%s", t.Feature, t.Action)
```

Reply:
updated to use `/` :)